webapp/packages/plugin-data-viewer/src/DataViewerSettingsService.ts

```diff
@@ -21,14 +21,13 @@ import { schema, schemaExtra } from '@cloudbeaver/core-utils';
 
 import { DATA_EDITOR_SETTINGS_GROUP } from './DATA_EDITOR_SETTINGS_GROUP.js';
 
-const FETCH_MIN = 10;
+const FETCH_MIN = 100;
```

Comment:
Why 100? Cant see this in ticket

Reply:
The case: We have 11k rows in the table and set the fetch size to 10. Now, we cannot load other data because the data loads in lazy mode when we scroll. Probably we should add a button for this case. But for now I set minimum size for 100 so user is not confused until we solve this issue

![Снимок экрана 2024-11-05 в 11 45 59](https://github.com/user-attachments/assets/a84989e1-a576-4edc-aae0-ff686b9bb1d1)